client/asset/btc/btc.go

```diff
@@ -3059,13 +3113,9 @@ func (btc *baseWallet) LocktimeExpired(contract dex.Bytes) (bool, time.Time, err
 // This method blocks until the redemption is found, an error occurs or the
 // provided context is canceled.
 func (btc *baseWallet) FindRedemption(ctx context.Context, coinID, _ dex.Bytes) (redemptionCoin, secret dex.Bytes, err error) {
-	exitError := func(s string, a ...interface{}) (dex.Bytes, dex.Bytes, error) {
-		return nil, nil, fmt.Errorf(s, a...)
-	}
```

Comment:
Doesn't add anything IMO.  It just makes me look up at what `exitError` is instead of just seeing right at the `return` what it is.  I think this pattern also leads us to not wrap errors for this reason - we could use %w but we don't because we're not giving it directly to `fmt.Errorf` it it feels weird, I guess.

I understand the purpose of this kind of thing in certain cases where you want to be certain to zero out the returns on error, esp. when using named return parameters, but I don't sense we're getting that benefit here.